load-responsively.js

```diff
@@ -32,13 +32,15 @@
       $container.css({ opacity: 0, transition: 'opacity 1s ease-in-out' });
       $this.css({position: 'absolute', top: 0, left: 0, width: '100%'});
 
-      if ($image.width !== undefined && $image.width !== 0) {
+      $image.on('load', function() {
         $container.css({ opacity: 1 });
-      }
-      else {
-        $image.on('load', function() {
-          $container.css({ opacity: 1 });
-        });
+        //TODO: remove this line and replace with onImageLoaded callback
+        $image.css({visibility: 'visible'});
+      });
+      if ($image[0].complete) {
+        setTimeout(function() {
+          $image.load();
```

Comment:
I don't think this is the correct thing here. (http://api.jquery.com/load/).  Not sure why it works, but I think you want `$image.trigger('load');`